src/actions/PetitionsActions.js

```diff
@@ -10,23 +10,34 @@ import {
 } from './actionTypes';
 
 export function fetchPetitions ({ location, params }) {
-  // Get query from react-router locatiin
+  // Get query from react-router location
   const { query } = location;
 
-  // Construct our query params, based on
-  // route params or query string params
+  // Construct our query params based on route params or query string params
   const queryParams = {
     page: parseInt(params && params.page || query.page || 1),
```

Comment:
Instead of always checking `params && params.something`, you can assign `params` a default value. 

```
export function fetchPetitions ({ location, params = {} ) { }
```

Makes for slighly DRY-er boolean checks.